docs/src/main/asciidoc/transaction.adoc

```diff
@@ -5,13 +5,16 @@ https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
 = Using Transactions in Quarkus
 include::_attributes.adoc[]
-:categories: data
-:summary: Quarkus comes with a Transaction Manager and uses it to coordinate and expose transactions to your applications. Each extension dealing with persistence will integrate with it for you, and you will explicitly interact with transactions via CDI. This guide will walk you through all that.
+:categories: data,transactions
 
-Quarkus comes with a Transaction Manager and uses it to coordinate and expose transactions to your applications.
-Each extension dealing with persistence will integrate with it for you.
-And you will explicitly interact with transactions via CDI.
-This guide will walk you through all that.
+Quarkus includes a Transaction Manager extension used to coordinate and expose transactions to your applications as described in the link: https://jakarta.ee/specifications/transactions/[Jakarta Transactions] specification, formerly known as Java Transaction API (JTA).
+
+When discussing Quarkus transactions, this guide refers to Jakarta Transactions transaction style and uses only the term _transaction_ to address them.
+
+Also, Quarkus does not support distributed transactions.
+This means that models that propagate transaction context, such as JTS Topology Suite (JTS), JBoss Remoting, REST-AT, XTS, and others, are not supported by the `narayana-jta` extension.
```

Comment:
My bad, I should have specified that JTS stands for [Java Transaction Service](https://download.oracle.com/otndocs/jcp/7309-jts-1.0-spec-oth-JSpec/)

Reply:
Can we remove `JBoss Remoting` from the list? We should also replace XTS with `WS-Atomic Transaction`.